.github/workflows/docker-publish.yaml

```diff
@@ -69,47 +64,48 @@ jobs:
           username: wildmeorg
           password: ${{ secrets.WBIA_WILDMEBOT_DOCKER_HUB_TOKEN }}
 
-      # Push tagged image (version tag + latest) to registries
+      # Push tagged image to registries
       - name: Tagged Docker Hub
         if: ${{ github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags/v') }}
         run: |
           VERSION=$(echo ${GITHUB_REF} | sed 's#.*/v##')
           echo "IMAGE_TAG=${VERSION}" >> $GITHUB_ENV
 
-      # Push bleeding-edge image (main tag) to registries
+      # Push bleeding-edge image to registries
       - name: Bleeding Edge Docker Hub
         if: github.ref == 'refs/heads/main'
-        run: |
-          echo "IMAGE_TAG=main" >> $GITHUB_ENV
+        run: echo "IMAGE_TAG=main" >> $GITHUB_ENV
 
-      # Push nightly image (nightly tag) to registries
+      # Push nightly image to registries
       - name: Nightly Docker Hub
         if: github.event_name == 'schedule'
-        run: |
-          echo "IMAGE_TAG=nightly" >> $GITHUB_ENV
+        run: echo "IMAGE_TAG=nightly" >> $GITHUB_ENV
 
-      # Build images
+      # Build and push images
       - name: Build Codex
```

Comment:
Kind of trivial ... but since this file is being touched, can we update this line to `- name: Build Scoutbot`?